packages/wrangler/package.json

```diff
@@ -36,9 +36,12 @@
     "cli"
   ],
   "dependencies": {
+    "@sentry/integrations": "^6.17.4",
+    "@sentry/node": "^6.17.4",
     "esbuild": "0.14.14",
     "miniflare": "2.2.0",
     "path-to-regexp": "^6.2.0",
+    "prompts": "^2.4.2",
```

Comment:
This is a stopgap. Should reimplement with Ink or find an alternative that uses Ink

Reply:
I think these dependencies can be moved to devDeps since we will be bundling them into the wrangler distributable, right?